bin/sql-review/cmd/root.go

```diff
@@ -0,0 +1,142 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/bytebase/bytebase/common"
+	"github.com/bytebase/bytebase/common/log"
+	server "github.com/bytebase/bytebase/sql-review-server"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	// Register pingcap parser driver.
+	_ "github.com/pingcap/tidb/types/parser_driver"
+	// Register fake advisor.
+	_ "github.com/bytebase/bytebase/plugin/advisor/fake"
+	// Register mysql advisor.
+	_ "github.com/bytebase/bytebase/plugin/advisor/mysql"
+	// Register postgresql advisor.
+	_ "github.com/bytebase/bytebase/plugin/advisor/pg"
+
+	// Register postgres parser driver.
+	_ "github.com/bytebase/bytebase/plugin/parser/engine/pg"
+)
+
+// -----------------------------------Global constant BEGIN----------------------------------------.
+const (
+
+	// greetingBanner is the greeting banner.
+	// http://patorjk.com/software/taag/#p=display&f=ANSI%20Shadow&t=Bytebase%20SQL%20Review
+	greetingBanner = `
+██████╗ ██████╗     ███████╗ ██████╗ ██╗         ██████╗ ███████╗██╗   ██╗██╗███████╗██╗    ██╗
+██╔══██╗██╔══██╗    ██╔════╝██╔═══██╗██║         ██╔══██╗██╔════╝██║   ██║██║██╔════╝██║    ██║
+██████╔╝██████╔╝    ███████╗██║   ██║██║         ██████╔╝█████╗  ██║   ██║██║█████╗  ██║ █╗ ██║
+██╔══██╗██╔══██╗    ╚════██║██║▄▄ ██║██║         ██╔══██╗██╔══╝  ╚██╗ ██╔╝██║██╔══╝  ██║███╗██║
+██████╔╝██████╔╝    ███████║╚██████╔╝███████╗    ██║  ██║███████╗ ╚████╔╝ ██║███████╗╚███╔███╔╝
+╚═════╝ ╚═════╝     ╚══════╝ ╚══▀▀═╝ ╚══════╝    ╚═╝  ╚═╝╚══════╝  ╚═══╝  ╚═╝╚══════╝ ╚══╝╚══╝
+
+%s
+___________________________________________________________________________________________
+
+`
+	// byeBanner is the bye banner.
+	// http://patorjk.com/software/taag/#p=display&f=ANSI%20Shadow&t=BYE
+	byeBanner = `
+██████╗ ██╗   ██╗███████╗
+██╔══██╗╚██╗ ██╔╝██╔════╝
+██████╔╝ ╚████╔╝ █████╗
+██╔══██╗  ╚██╔╝  ██╔══╝
+██████╔╝   ██║   ███████╗
+╚═════╝    ╚═╝   ╚══════╝
+
+`
+)
+
+// -----------------------------------Command Line Config BEGIN------------------------------------.
+var (
+	flags struct {
+		// Used for Bytebase command line config
+		host  string
+		port  int
+		debug bool
+	}
+	rootCmd = &cobra.Command{
+		Use:   "sql-review",
```

Comment:
How about calling it "sql"?

./sql is easier to memorize than ./sql-review

And we can call the service SQL Service instead of SQL Review Service

Reply:
+1 and we may extend it to do other none sql review API stuff in the future.